README.md

```diff
@@ -204,6 +224,24 @@ Linux* kernel                     | *N/A*          | 4.11 or higher | *N/A*
             <td> NVIDIA CUDA SDK </td>
             </tr>
         </tr>
+        <tr>
+            <td rowspan=8> Windows* </td>
+            <tr>
+                <td rowspan=2> Intel CPU </td>
+                <td> Intel(R) oneAPI DPC++ Compiler <br> or <br> Intel project for LLVM* technology </td>
+                <tr>
+                    <td> Intel(R) oneAPI Math Kernel Library </td>
+                </tr>
+            </tr>
+            <td rowspan=3> Intel GPU </td>
+            <td> Intel(R) oneAPI DPC++ Compiler </td>
+            <tr>
+                <td> Intel GPU driver </td>
+            </tr>
+            <tr>
+                <td> Intel(R) oneAPI Math Kernel Library </td>
+            </tr>
+        </tr>    
```

Comment:
I think we can also use "Linux/Windows" for Intel CPU and GPU in OS column, and "Linux" for NVIDIA GPU, and remove duplication.

Reply:
Please, check updated version